roles/ceph-grafana/templates/grafana-server.service.j2

```diff
@@ -21,6 +21,7 @@ ExecStart=/usr/bin/{{ container_binary }} run --rm --name=grafana-server \
   --memory={{ grafana_container_memory }}GB \
   --memory-swap={{ grafana_container_memory * 2 }}GB \
   -e GF_INSTALL_PLUGINS={{ grafana_plugins|join(',') }} \
+  {{ ceph_grafana_docker_extra_env }} \
```

Comment:
I think we should avoid using docker for new variables name and use a more generic word like container.

Also even if it's not the initial purpose of this PR but I think we should do the same for prometheus right ?

Reply:
I tried to follow the current convention (`ceph_mon_docker_extra_env`, `ceph_osd_docker_extra_env`, etc.), but I can change it to `ceph_grafana_container_extra_env` if you prefer.

I can do the same for `prometheus` and `alertmanager`.